Src/RhinoPlugin.Tests.Xunit/XunitTestFixture.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Microsoft.Win32;
+using System;
+using System.IO;
+using System.Reflection;
+using Xunit;
+
+namespace RhinoPlugin.Tests.Xunit
+{
+
+    /// <summary>
+    /// Shared test context across unit tests that loads rhinocommon.dll and grasshopper.dll
+    /// </summary>
+    public class XunitTestFixture : IDisposable
+    {
+        bool initialized = false;
+        static string rhinoDir;
+        Rhino.Runtime.InProcess.RhinoCore _rhinoCore;
```

Comment:
Suggest we tidy these up a bit:
```suggestion
        private bool _initialized = false;
        private string _rhinoDir;
        private Rhino.Runtime.InProcess.RhinoCore _rhinoCore;
```

Which means we also need to make `ResolveGrasshopper` into an instance member as well